### ISSUE TITLE
Incorrect links in the playbook.md file

### ISSUE THREAD

**Issue by etolstoy** [artsy/mobile#92]
Hi!

Sorry for rushing into your task tracker. I've been reading your playbook for some inspiration (I want to do the same in our mobile team) and encountered two broken links. 

The first one is *Artsy/Mobile*:

> Current members of the Mobile Practice can be found on the Artsy/Mobile index of our GitHub repo.

The second one is *artsy/potential*:

> Most of the general Artsy Developer information for onboarding is covered in artsy/potential.

And thanks for your work with the community!

**Comment by ashfurrow** [comment]
Yikes! Thanks for letting us know. The first link is just to https://github.com/artsy/mobile (I'll fix that). 

The artsy/potential link has unfortunately been closed by the broader engineering team. A repository containing info on all our web services exposed some business logic that posed a business risk; after speaking with the lead web engineer, we concluded that having a single source of truth for Artsy engineers was important, and that a "partially closed" solution didn't really exist, so we had to close it. I think a note that the potential repo is private should be added (I'll do that too).

Never worry about opening an issue on this repo, we have it here and open so that developers can ask us questions and offer suggestions for our workflow. If there's anything else I can help with, let me know!
